package.json

```diff
@@ -12,19 +12,22 @@
         "@adobe/asset-compute-events-client": "^1.0.1",
         "@adobe/jwt-auth": "^0.3.1",
         "@adobe/node-fetch-retry": "^1.0.1",
+        "fs-extra": "^9.0.1",
+        "js-yaml": "^3.14.0",
+        "json-to-pretty-yaml": "^1.2.2",
```

Comment:
looks like `json-to-pretty-yaml` is not used anywhere. I think we can remove this